src/manifest/dbtProject.ts

```diff
@@ -958,7 +959,54 @@ select * from renamed
     }
     this.queryResultPanel.getSummary(compiledSql, this.adapterType);
   }
+  private getLimitQuery(queryTemplate: string, query: string, limit: number) {
+    return queryTemplate
+      .replace("{query}", () => query)
+      .replace("{limit}", () => limit.toString());
+  }
+  private async getQuery(
+    query: string,
+    limit: number,
+  ): Promise<{ queryTemplate: string; limitQuery: string }> {
+    const queryTemplate = workspace
+      .getConfiguration("dbt")
+      .get<string>("queryTemplate");
+
+    if (queryTemplate && queryTemplate !== DefaultQueryTemplate) {
```

Comment:
Why do we explicitly check if `!==  DefaultQueryTemplate`, If users set it why don't we just use it? Isn't it null in case of unset, otherwise why do we need to put it in the other place as 

```
const queryTemplate = workspace
      .getConfiguration("dbt")
      .get<string>("queryTemplate", DefaultQueryTemplate);
```

Reply:
@mdesmet i also assumed that we will get null, if this value is not set. but i am getting default value that is set in package.json. Because of that i had to add this check

Reply:
my bad. I am actually getting empty value now. fixed the code.